_data/events/virtual-2020-11-06-global-day-of-coderetreat-singapore-2020.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Global Day of CodeRetreat Singapore 2020",
+  "format": "classic",
+  "spoken_language": "English",
+  "url": "https://www.meetup.com/Junior-Developers-Singapore/events/274176954",
+  "date": {
```

Comment:
Your meetup event is for the 7th, this payload is for the 6th. Could you fix this please before we merge it?

Reply:
Sure. Updated. Thanks!